shared-ui/components/meet-the-team/meet-all-core/MeetAllCore.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { min } from '../../../lib/responsive';
+import useMatchMedia from 'react-use-match-media';
+import {
+  StyledMeetCoreContainer,
+  StyledOurTeamCore,
+  StyledHeaderCore,
+  StyledDescriptionTeamCore,
+  StyledImageCore
+} from './MeetAllCore.styles';
+import GroupDesktop from '../../../images/meet-the-team/groupDesktop.png';
+import GroupMobile from '../../../images/meet-the-team/groupMobile.png';
```

Comment:
On the mobile version now the image on mobile is the same as desktop so we can just get rid of the mobile image
<img width="611" alt="Screenshot 2022-11-19 at 12 51 31 PM" src="https://user-images.githubusercontent.com/59738880/202864628-aed5ea84-2910-41e2-99d0-9ba76e8f6ca1.png">